.do/app.yaml

```diff
@@ -19,7 +19,7 @@ services:
       repo: chocolat-chaud-io/stator
       branch: master
       deploy_on_push: true
-    build_command: npm run remap-redoc && npm run typeorm -- migration:run && npm run build api -- --prod
```

Comment:
we don't need to run migrations anymore?

Reply:
That's what was causing the issue. We will need to figure something out for this as the env variables are not available a build time...

Reply:
Yeah makes sense. anyway auto migration is on by default right

Reply:
maybe you could somehow add it to the start command?

Reply:
Yeah, I guess that would be possible 🤔